src/otx/recipe/detection/atss_mobilenetv2.yaml

```diff
@@ -115,7 +115,7 @@ model:
           - 255
           - 255
           - 255
-        bgr_to_rgb: true
+        bgr_to_rgb: false
```

Comment:
Should this value be always false or follow original mmX config?

Reply:
After this PR, our data pipeline used the PIL.load(RGB) to load the image, however, mmX used the openCV(BGR). So we need to change the value according to the loading pipeline.

Reply:
> After this PR, our data pipeline used the PIL.load(RGB) to load the image, however, mmX used the openCV(BGR). So we need to change the value according to the loading pipeline.

True

Reply:
So it is reverse of original mmX config right?

Reply:
Yes, if mmX will use the openCV loading.

Reply:
> So it is reverse of original mmX config right?

Yes